### PR TITLE
Update flutterinfinitelisttutorial.md

### PR DESCRIPTION
**Pull request by oozd**
Finished PostBloc missing declaration of currentState.

## Status
**READY**

## Breaking Changes
NO

## Description
Mistake on docs: in infiniteList example, finished PostBloc example is missing declaration of currentState.

## Related PRs
None

branch | PR

## Impact to Remaining Code Base

*

### CODE REVIEW

File: docs/flutterinfinitelisttutorial.md [hunk 1]
```diff
@@ -347,6 +347,7 @@ class PostBloc extends Bloc<PostEvent, PostState> {
 
   @override
   Stream<PostState> mapEventToState(event) async* {
+  final currentState = state;
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md]: can you please fix the indentation? Also we should update all of the translated `flutterinfinitelisttutorial.md` files within each of the subdirectories.

File: docs/flutterinfinitelisttutorial.md [hunk 2]
```diff
@@ -347,6 +347,7 @@ class PostBloc extends Bloc<PostEvent, PostState> {
 
   @override
   Stream<PostState> mapEventToState(event) async* {
+    final currentState = state;
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md]: can we also make this change to all of the translated `flutterinfinitelisttutorial.md` files?

**Comment by oozd** [comment]
All documents fixed.

**Comment by oozd** [comment]
Accidentally closed, reopened. All documents fixed.

**Pull request merged**